src/components/metadata/metadata.js

```diff
@@ -107,6 +109,27 @@ const MetaData = ({
       : value?.split('.').pop();
   };
 
+  const statsContent = (statValue, statLabel) => {
```

Comment:
Optional: move this to its own component so this function component doesn't get created on every time it metadata components changed or renders.